src/types/motions.ts

```diff
@@ -84,3 +85,12 @@ export interface MotionQuery {
   motionData: MotionData;
   createdAt: string;
 }
+
+export interface Event {
+  name: string;
+  transactionHash: string;
+  logIndex: number;
```

Comment:
@rdig logIndex is always zero. Is this expected as I am just adding this to create the key in the ActionDetailsPageFeed?

Reply:
@Julianahlk 
LogIndex is always 0 for you due to the nature of the local chain, which just sends one transaction at a time, meaning your events will always have index 0 locally, however in production this number will almost always be non-zero due to multiple transactions and events being handled inside a single block. So your combination of `<transactionHash>_<logIndex>` will always be unique, both locally and in production

@willm30 

No, since multiple events can be emmited inside a single block, meaning the IDs will be duplicated